src/scss/theme/_button.scss

```diff
@@ -2,7 +2,7 @@
 
 // this will only pull in the original tailwind from node_modules, not our custom version
 @import '../utilities/index';
-
+.button, // special override for user-login page
```

Comment:
I added this `.button` class since the class being added to the local login button does not contain any of our `.su-button` variations

Reply:
Can you double check? I see su-button and that's why i'm targeting it on lines 36-38 below (but I only targetted the hover/focus because it had picked up the regular default button style for me.

Reply:
If they have local Drupal login configured, the login form generated `page.content` (see above image) contains an input button that does not use our `.su-button` variation.

Here's a sample snippet of the button:
```<input data-drupal-selector="edit-submit" type="submit" id="edit-submit" name="op" value="Log in" class="button js-form-submit form-submit">```

Reply:
I can move the styles (along with the other `.system-main-block` styles) into the user-login page as the classes are unique to this page :)